projects/igniteui-angular/src/lib/tree/tree.component.ts

```diff
@@ -504,6 +504,11 @@ export class IgxTreeComponent implements IgxTree, OnInit, AfterViewInit, OnDestr
         requestAnimationFrame(() => {
             this.selectionService.selectNodesWithNoEvent(toBeSelected);
         });
+        if(this.platform.isBrowser) {
```

Comment:
Compilation fails: 

✖ Compiling with Angular sources in partial compilation mode.
projects/igniteui-angular/src/lib/tree/tree.component.ts:507:17 - error TS2339: Property 'platform' does not exist on type 'IgxTreeComponent'.

Reply:
added the missing import and removed the extra raf